src/parse.js

```diff
@@ -204,7 +204,20 @@ class VCF {
    * CRLF newlines.
    */
   parseLine(line) {
-    const fields = line.trim().split('\t')
+    line = line.trim()
+    if (!line.length) return undefined
+    let t = 0
+    let i = 0
+    for (; i < line.length; i += 1) {
+      if (line[i] === '\t') {
+        t += 1
+      }
+      if (t === 9) {
+        break
+      }
+    }
+    const fields = line.substr(0, i).split('\t')
+    const rest = line.substr(i + 1).split('\t')
```

Comment:
Since sample info is evaluated lazily, we can speed this up by moving the split into `_parseGenotypes()`. I get a speedup of ~450 ms to about ~130 ms in the `console.time` test with this change.
```suggestion
    const rest = line.substr(i + 1)
```

Reply:
oh nice, that's the ticket...